app/src/main/java/chat/rocket/android/app/RocketChatApplication.kt

```diff
@@ -99,4 +102,11 @@ class RocketChatApplication : Application(), HasActivityInjector, HasServiceInje
     override fun serviceInjector(): AndroidInjector<Service> {
         return serviceDispatchingAndroidInjector
     }
+
+    companion object {
+        var context: WeakReference<Context>? = null
+        fun getAppContext(): Context? {
+            return context?.get()
+        }
+    }
```

Comment:
I understand why you put this but we can use Dagger instead. I think will do for now :)